package.json

```diff
@@ -60,44 +60,44 @@
     }
   },
   "devDependencies": {
-    "@babel/core": "^7.23.3",
-    "@babel/plugin-proposal-async-generator-functions": "^7.20.7",
-    "@babel/plugin-transform-modules-commonjs": "^7.23.3",
-    "@babel/plugin-transform-runtime": "^7.23.3",
-    "@babel/polyfill": "^7.2.5",
-    "@babel/preset-env": "^7.23.3",
-    "@babel/preset-react": "^7.23.3",
-    "@babel/register": "^7.22.15",
+    "@babel/core": "^7.27.1",
+    "@babel/plugin-transform-async-generator-functions": "^7.27.1",
```

Comment:
The `@babel/plugin-proposal-async-generator-functions` package is deprecated with the following message:

> This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.